<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prete</class>
 <widget class="QDialog" name="prete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saisir le prete</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>19</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Numero de pret</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="numeroDePret"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Numero du lecteur</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="numeroLecteur"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Numero du Livre</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="numeroLivre"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date de pret</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Date de retour</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="datePret"/>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="dateRetour"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="SaisirLePrete">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:green;
color:white;
font-weight:bold;
}</string>
   </property>
   <property name="text">
    <string>Saisir le Prete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:red;
color:white;
font-weight:bold;
}</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
